frontend/src/domain/Account/PersonalData/PersonalData.tsx

```diff
@@ -0,0 +1,187 @@
+                              name="apartment"
+                              initialValue={mockData.apartment}
+                              parse={(value) => value ? normalizeToNumber(value) : value}
+                              validate={fieldRequired}
+                              render={({ input, meta }) => (
+                                 <P.InputWrapper>
+                                    <P.Label>Apartment number</P.Label>
+                                    <P.StyledInput {...input} disabled={!isEditEnable} />
+                                    {meta.touched && meta.error && <P.ValidationError>{meta.error}</P.ValidationError>}
+                                 </P.InputWrapper>
+                              )}
+                           />
+                           <Field
+                              name="code"
+                              initialValue={mockData.code}
+                              parse={(value) => value ? normalizeToZipCode(value) : value}
+                              validate={basicValidation}
+                              render={({ input, meta }) => (
+                                 <P.InputWrapper>
+                                    <P.Label>Zip code</P.Label>
+                                    <P.StyledInput {...input} disabled={!isEditEnable} />
+                                    {meta.touched && meta.error && <P.ValidationError>{meta.error}</P.ValidationError>}
+                                 </P.InputWrapper>
+                              )}
+                           />
+                           <Field
+                              name="city"
+                              initialValue={mockData.city}
+                              validate={basicValidation}
+                              render={({ input, meta }) => (
+                                 <P.InputWrapper>
+                                    <P.Label>City</P.Label>
+                                    <P.StyledInput {...input} disabled={!isEditEnable} />
+                                    {meta.touched && meta.error && <P.ValidationError>{meta.error}</P.ValidationError>}
+                                 </P.InputWrapper>
+                              )}
+                           />
+                           <Field
+                              name="country"
+                              initialValue={mockData.country}
+                              validate={basicValidation}
+                              render={({ input, meta }) => (
+                                 <P.InputWrapper>
+                                    <P.Label>Country</P.Label>
+                                    <P.StyledInput {...input} disabled={!isEditEnable} />
+                                    {meta.touched && meta.error && <P.ValidationError>{meta.error}</P.ValidationError>}
+                                 </P.InputWrapper>
+                              )}
+                           />
```

Comment:
There are a couple of almost the same parts of the code, maybe we can iterate through it to have it more DRY e.g. 
```
const fieldsList = [
{
 name: "country",
validationCallback: basicValidation,
}
...
];

...
 {fieldsList.map(({ name, validationCallback }) => (
            <Field
              name={name}
              initialValue={mockData[name]}
              validate={validationCallback}
              render={({ input, meta }) => (
                <P.InputWrapper>
                  <P.Label>{name}</P.Label> //css: text-transform:capitalize;
                  <P.StyledInput {...input} disabled={!isEditEnable} />
                  {meta.touched && meta.error && (
                    <P.ValidationError>{meta.error}</P.ValidationError>
                  )}
                </P.InputWrapper>
              )}
            />
          ))}
```

Reply:
done, but text-transform:capitalize; is not good enough, because sometimes field have a two/three words label